CONFIGDOC.md

```diff
@@ -480,7 +616,7 @@ If you define your pipeline using scripted pipeline syntax, use the `tool` keywo
 node {
     def matlabver
     stage('Run MATLAB Command') {
-        matlabver = tool 'R2024a'
+        matlabver = tool 'R2024b'
         if (isUnix()) {
             env.PATH = "${matlabver}/bin:${env.PATH}"   // Linux or macOS agent
         } else {

```

Comment:
Is it really true that the plugin doesn't automatically add MATLAB to path for a scripted pipeline?